api/users/views.py

```diff
@@ -17,13 +17,21 @@ class UserMixin(object):
     node_lookup_url_kwarg = 'user_id'
 
     def get_user(self, check_permissions=True):
+
+        key = self.kwargs[self.node_lookup_url_kwarg]
+        if key == 'me':
+            return self.request.user
+        elif check_permissions:
+            obj = get_object_or_404(User, self.kwargs[self.node_lookup_url_kwarg])
+            self.check_object_permissions(self.request, obj)
+            return obj
+
         obj = get_object_or_404(User, self.kwargs[self.node_lookup_url_kwarg])
         if check_permissions:
             # May raise a permission denied
             self.check_object_permissions(self.request, obj)
         return obj
 
-
```

Comment:
Classes should have two blank lines between them.

ref: https://www.python.org/dev/peps/pep-0008/#id15